code/parachain/frame/staking-rewards/src/lib.rs

```diff
@@ -134,7 +137,7 @@ pub mod pallet {
 			/// FNFT Instance Id
 			fnft_instance_id: T::FinancialNftInstanceId,
 			/// Reward multiplier
-			reward_multiplier: Perbill,
```

Comment:
Perbill was nice type. it told what is usage for this - it is from 0 to 1 and with step of 1/10^12. While FixedU64 has no restrictions but is used as from 0 to 1 with step of 1/10^2.

You may type MyPrecentage = FixedU64 so we clear what it is about?

Also there Permill and Percentage types too afaik in sp. Also some just used 1 to 2^bits types instead of decimal percentages.

What I want to say that not sure if current switch is better against what it was.